_data/pe/docs-home.yml

```diff
@@ -20,6 +20,8 @@ toc:
     path: /docs/pe/solution-templates/fleet-tracking/
   - title: Water metering
     path: /docs/pe/solution-templates/water-metering/
+  - title: Air quality monitoring
```

Comment:
The same link should be present when I browse: https://thingsboard.io/docs/pe/ and https://thingsboard.io/docs/paas/